nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -312,7 +312,7 @@ def allreduce_first_last_embeddings(self):
         if parallel_state.get_pipeline_model_parallel_world_size() > 1 and (
             parallel_state.is_pipeline_first_stage() or parallel_state.is_pipeline_last_stage()
         ):
-            if self.model.share_word_embeddings:
+            if self.model.share_token_embeddings:
```

Comment:
I think we should revert this, unless we edited the GPT config file?

Reply:
I did not change it. However, `share_word_embeddings` does not appear in any yaml or python file in nemo.

Reply:
Fixed Jenkinsfile.